pkg/server/server_test.go

```diff
@@ -131,6 +140,29 @@ func TestHealthzHandler(t *testing.T) {
 	}
 }
 
+func TestCancelHandler(t *testing.T) {
+	s := initServer()
+
+	testBody := strings.NewReader("{\"Filter\":{\"id\":1}}")
+
+	req, err := http.NewRequest("POST", "/cancel", testBody)
+
```

Comment:
No space before `if err...`

Reply:
Is there a way we can add this as a linting rule? It's the best way to force coding standardization - otherwise when new folks hop onto the project we might see the same thing if they're not familiar with Go (like myself).

Reply:
Good question. I am unsure, but this is worth looking into.